Overview.src.html

```diff
@@ -160,7 +160,6 @@ <h2 class="short">Conformance</h2>
  <li><dfn>process request body</dfn>
  <li><dfn>process request end-of-file</dfn>
  <li><dfn>process response</dfn>
- <li><dfn>process response body</dfn>
```

Comment:
I checked and HTML does not use this, but maybe other people do? I guess it's probably OK to remove.